src/Microsoft.IdentityModel.Protocols/Configuration/HttpDocumentRetriever.cs

```diff
@@ -44,6 +44,11 @@ public class HttpDocumentRetriever : IDocumentRetriever
         private HttpClient _httpClient;
         private static readonly HttpClient _defaultHttpClient = new HttpClient();
 
+#pragma warning disable 1591
```

Comment:
What warning is this disabling?

Reply:
Missing XML comment for publicly visible type or member 'Type_or_Member'

Reply:
Why not populate the XML comment?

Reply:
added XML